dotnet/driver.go

```diff
@@ -286,7 +283,7 @@ func NewDriver(ctx context.Context, logger hclog.Logger) drivers.DriverPlugin {
 
 	return &Driver{
 		eventer: eventer.NewEventer(ctx, logger),
-		config:  *new(Config),
+		config:  new(Config),
```

Comment:
Pointer is needed for CheckDotnetVersionInfo function to be able to set the SdkPath if it is empty